dll/win32/shell32/folders/CFindFolder.cpp

```diff
@@ -0,0 +1,609 @@
+#include "CFindFolder.h"
+#include <exdispid.h>
+#include "../shresdef.h"
+
+WINE_DEFAULT_DEBUG_CHANNEL(CFindFolder);
+
+// Declaration for function that is not fully implemented
+EXTERN_C HRESULT
+WINAPI
+SHOpenFolderAndSelectItems(LPITEMIDLIST pidlFolder,
+                           UINT cidl,
+                           PCUITEMID_CHILD_ARRAY apidl,
+                           DWORD dwFlags);
+
+struct FolderViewColumns
+{
+    LPCWSTR wzColumnName;
+    DWORD dwDefaultState;
+    int fmt;
+    int cxChar;
+};
+
+static FolderViewColumns g_ColumnDefs[] =
+{
+    {L"Name",      SHCOLSTATE_TYPE_STR | SHCOLSTATE_ONBYDEFAULT, LVCFMT_LEFT, 30},
```

Comment:
Very nice pull request! Is this thing localized?

Reply:
He'd have to port his Dialog to all other resource files. 
right now as far as i can see its only on EN-US

Reply:
And add resource strings for these items